src/customers/account-dashboard-approval-rules.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Approval Rules
+---
+
+Most companies have order approval rules for purchase orders. Companies can control who gets to create purchase orders and how much they can spend.
+For instance:
+
+*  Any PO less than X value is automatically approved.
+*  POs that is over X value but less than Q must be approved by Y.
+*  Any PO over X value must be approved by Y and Z.
+*  Anyone making a PO at the Director level or above are automatically approved.
+
+Magento lets you set up these rules to enforce company policies.
+
+To create rules, users with role permissions have an 'Approval Rules' link in the left nav bar.
+Permissions to create rules are set in the Roles and Permissions page in the left nav bar.
+
+If no rules have been created, the list is empty and the 'Create Rule' button is displayed.
+Click the button to create a new rule.
+
+On the Approval Rules form, rules can be Enabled or Disabled.
+
+Note: When assigning a role as an approver, ensure that there is at least one user in that role.
+
+## Rule types and common options
+
+Choose the rule type from the dropdown menu.
```

Comment:
"dropdown menu" is not OK in user docs. I don't know if there is a label here or what this looks like. If the dropdown has a label, use that name to refer to the UI element. 

```suggestion
Choose the **Rule Type**.
```